Source/Charts/Renderers/LineChartRenderer.swift

```diff
@@ -562,12 +562,12 @@ open class LineChartRenderer: LineRadarRenderer
 
         for i in 0 ..< dataSets.count
         {
-            guard let dataSet = lineData.getDataSetByIndex(i) as? ILineChartDataSet else { continue }
-            
-            if !dataSet.isVisible || dataSet.entryCount == 0
-            {
-                continue
-            }
+            guard
+                let dataSet = lineData.getDataSetByIndex(i) as? ILineChartDataSet,
+                dataSet.drawCirclesEnabled,
```

Comment:
👍

Reply:
The ` if !dataSet.isDrawCirclesEnabled` in the `for j in _xBounds` loop can then be removed.

Reply:
Good point! Updated the PR to fix it.